contents.json

```diff
@@ -9935,11 +9935,11 @@
       "itunes": "https://itunes.apple.com/app/tpg-offline/id1001560047",
       "screenshots":
       [
-           "http://a1.mzstatic.com/us/r30/Purple18/v4/53/d9/61/53d961c4-8ee4-03ef-a297-4b81c1aa1a53/screen696x696.jpeg",
-           "http://a1.mzstatic.com/us/r30/Purple60/v4/c5/16/fc/c516fc33-f75d-06eb-69fc-24354006a893/screen696x696.jpeg",
-           "http://a2.mzstatic.com/us/r30/Purple18/v4/3f/b5/77/3fb57762-723d-fe11-b6c7-8b99a07fe021/screen696x696.jpeg",
-           "http://a5.mzstatic.com/us/r30/Purple30/v4/73/9e/96/739e962c-5ef3-86e0-a445-454303da709f/screen696x696.jpeg",
-           "http://a2.mzstatic.com/us/r30/Purple30/v4/7f/1e/e5/7f1ee5dc-0640-7b76-4969-f135894dc9d8/screen696x696.jpeg"
+           "https://github.com/RemyDCF/tpg-offline/blob/master/Screenshot/Stops.png",
```

Comment:
hey remy, thx for the update
however we need to link to the actual image, i.e. https://raw.githubusercontent.com/RemyDCF/tpg-offline/master/Screenshot/Stops.png

can you update all the links